web/src/muted_topics_ui.js

```diff
@@ -35,6 +39,9 @@ export function handle_topic_updates(user_topic) {
     const old_muted_topics = user_topics.get_muted_topics();
     user_topics.set_user_topic(user_topic);
     popover_menus.get_topic_menu_popover()?.hide();
+    // We need to destroy all the tippy instances from the DOM before re-rendering to
+    // prevent the appearance of tooltips whose reference has been removed.
+    tippyjs.destroyAllInstances();
```

Comment:
We should do this in `message_list.rerender` and `message_list_view.rerender_messages`.

Reply:
Sure, I have moved the function call inside `message_list.rerender` and `message_list_view.rerender_messages`. However, for `mute/unmute` streams, neither of these functions are called, so the `instances` are not destroyed. To handle this edge case, I have added the `destroyAllInstances` function call inside `stream_muting.update_is_muted`, since it is called every time the stream is `muted/unmuted`. Please let me know if there is a more suitable location for this function call.

Reply:
`message_list.rerender` is called in that case from inside `update_muting_and_rerender`. Can you verify again without having `destroyAllInstances` inside `stream_muting`.

Reply:
Yup just removed it, pushing the changes.